pkg/ocm/policy.go

```diff
@@ -179,6 +216,45 @@ func (builder *PolicyBuilder) Update(force bool) (*PolicyBuilder, error) {
 	return builder, err
 }
 
+// WithRemediationAction sets a RemediationAction in the policy definition.
+func (builder *PolicyBuilder) WithRemediationAction(action policiesv1.RemediationAction) *PolicyBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
```

Comment:
Please add log, something like:
glog.V(100).Infof("Creating policy %s in namespace %s with RemediationAction: %v",